src/stdbuf/stdbuf.rs

```diff
@@ -1,5 +1,4 @@
 #![crate_name = "stdbuf"]
-#![feature(negate_unsigned)]
```

Comment:
Not sure if this util should simply be removed. As fas as I can tell it will only ever work with glibc based binaries and is the only one that introduces C code/lib.